backend/src/routes/api/storage-class/utils.ts

```diff
@@ -23,10 +23,15 @@ export async function updateStorageClassConfig(
       'storageclasses',
       params.storageClassName,
     );
-
-    // Extract and update the annotations
-    const annotations = (storageClass.body as K8sResourceCommon).metadata.annotations || {};
-    annotations['opendatahub.io/sc-config'] = JSON.stringify(body);
+    const existingAnnotations = (storageClass.body as K8sResourceCommon).metadata.annotations || {};
+    const storageClassAnnotation = {
+      'opendatahub.io/sc-config': body
+        ? JSON.stringify({
+            ...JSON.parse(existingAnnotations['opendatahub.io/sc-config'] || null),
```

Comment:
If the existing annotation exists, it should be of string format, and if it doesn't exist, parsing null leads to a null value, and spreading null into an object leads to an empty object. Do you see an issue with this?

Reply:
if the string is not valid json it would cause an issue

```js
JSON.parse("invalid")
"Uncaught SyntaxError: Unexpected token i in JSON at position 0"
```

Reply:
Good point. I will address that.